packages/sw-runtime-caching/src/lib/request-wrapper.js

```diff
@@ -194,7 +194,7 @@ class RequestWrapper {
     } catch (err) {
       if (this.pluginCallbacks.fetchDidFail) {
         for (let callback of this.pluginCallbacks.fetchDidFail) {
-          callback({request});
+          callback({request: clonedRequest});
```

Comment:
yeah i had this exact idea if you see one of my commits in this PR, then was conflicted..
but yeah `callback({request: clonedRequest.clone()});` makes perfect sense
will do